src/scrab-bag.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+export default function ScrabBag(input) {
+  let scrabbleLetters = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3, 
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+    _: 2
+  }
+
+  let array = input.toUpperCase().split(' ')
+  for(let i = 0; i < array.length; i++) {
+    let currentLetters = array[i]
+    let remainderPieces = currentLetters[array]
+    remainderPieces = remainderPieces - 1
+  }
+  return scrabbleLetters
```

Comment:
`scrabbleLetters` was never altered. You are simply returning the array to declared at the top of the function. You will need to use the input to modify the array.